src/fidesops/service/connectors/query_config.py

```diff
@@ -242,10 +261,15 @@ class SQLQueryConfig(QueryConfig[TextClause]):
 
     def format_fields_for_query(
         self,
-        fields: List[str],
+        field_paths: List[FieldPath],
     ) -> List[str]:
-        """Returns fields in a format they can be added into SQL queries."""
-        return fields
+        """Returns field pathspaths in a format they can be added into SQL queries.
+
+        This currently takes no nesting into account and only returns the
+        last value from each key. It will need to be updated to support
+        nested values.
+        """
+        return [fk.levels[-1] for fk in field_paths]
```

Comment:
Am I correct in thinking that in the event the field isn't nested, `fk.levels[-1]` will return the only element in the list?

Reply:
exactly, yes:

```python
>>> fruits: Tuple[str] = ("apples", "bananas", "oranges")
>>> fruits[-1]
'oranges'
>>> vegetables = ("carrots",)
>>> vegetables[-1]
'carrots'
```